app/components/views/SettingsPage/SettingsPage.jsx

```diff
@@ -25,10 +25,19 @@ const closeWalletWithAutobuyerModal = (walletName) => (
   />
 );
 
+const closeWalletWithUnpaidFeeModal = (walletName) => (
+  <T
+    id="settings.closeWalletModalWithUnpaidFeeModal"
+    m="Are you sure you want to close {walletName} and return to the launcher? You still have unpaid tickets fee. If you proceed and they are chosen to vote, they will be missed."
```

Comment:
We probably need some more information here - what should the user do if they see this modal?

- Do they need to press a button somewhere?
- Do they just need to wait a bit longer? - 1 minute? Wait for a block to be mined?

Reply:
Good point, Let's change it to the following.

"Please go back to the ticket status page and press the Sync VSP Tickets button to ensure that all of your ticket fees are paid."